apis/keda/v1alpha1/scaletriggers_types.go

```diff
@@ -46,6 +46,10 @@ type AuthenticationRef struct {
 	// Kind of the resource being referred to. Defaults to TriggerAuthentication.
 	// +optional
 	Kind string `json:"kind,omitempty"`
+	// FilePath specifies a file containing auth parameters as JSON map[string]string.
+	// When set, auth params are read directly from this file instead of fetching TriggerAuthentication.
+	// +optional
+	FilePath string `json:"filePath,omitempty"`
```

Comment:
this is imho not the right place for the specification of the file path, I would rather see this under `TriggerAuthenticationSpec`
https://github.com/kedacore/keda/blob/bc5460f6946a2f984a13217b93ecdb9ce950c416/apis/keda/v1alpha1/triggerauthentication_types.go#L74-L101